network/rtpdump.ksy

```diff
@@ -0,0 +1,73 @@
+meta:
+  id: rtmpdump
+  title: Rtpdump (rtptools)
+  file-extension: rtmpdump
+  license: Unlicense
+  ks-version: 0.1
+  endian: be
+  imports:
+    - /network/rtp_packet
+doc: |
+  rtpdump is a format used by rtptools to record and replay
+  rtp data from network capture.
+doc-ref: https://chromium.googlesource.com/external/webrtc/stable/talk/+/master/media/base/rtpdump.h
+seq:
+  - id: file_header
+    type: header_t
+  - id: packets
+    type: packet_t
+    repeat: eos
+-includes:
+  - rtp_packet.ksy
+types:
+  header_t:
+    seq:
+      - id: shebang
+        contents: '#!rtpplay1.0'
+      - id: space
+        contents: ' '
+      - id: ip
+        type: str
+        encoding: ascii
+        terminator: 0x2f // '\'
+      - id: port
+        type: str
+        encoding: ascii
+        terminator: 0x0a // '\n'
+      - id: start_sec
+        type: u4
+        doc: |
+          start of recording, the seconds part.
+      - id: start_usec
+        type: u4
+        doc: |
+          start of recording, the microseconds part.
+      - id: ip2
+        type: u4
+        doc: |
+          network source.
+      - id: port2
+        type: u2
+        doc: |
+          port.
+      - id: padding
+        type: u2
+        doc: |
+          2 bytes padding.
+  packet_t:
+    seq:
+      - id: length
+        type: u2
+        doc: |
+          packet length (including this header).
+      - id: plen

```

Comment:
```suggestion
      - id: len_body
```

It's recommended to stick to the "standard" naming specified in the [KSY style guide] (http://doc.kaitai.io/ksy_style_guide.html#attr-id). For example, if some integer denotes the byte size of an attribute  `item` (i.e. this integer will be used in the `size` key of the `item`), it's recommended to name it `len_item`.